### PR TITLE
Make sure that generating pyapi does not throw errors

### PR DESCRIPTION
**Pull request by indie**
due to directory structure, or linenos

### CODE REVIEW

File: doc/sphinx/source/python_api/index.rst [hunk 1]
```diff
@@ -8,9 +8,9 @@ This section contains nGraph™ Python API documentation. Python API exposes
 nGraph™ C++ operations to Python users. For quick-start you can find an example
 of the API usage below.
 
-.. literalinclude:: /../../../python/examples/basic.py
+.. literalinclude:: ../../../../python/examples/basic.py
```
Comment (arogowie-intel) [inline doc/sphinx/source/python_api/index.rst]: According to [doc](http://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude): 
,, The file name is usually relative to the current file’s path. __However, if it is absolute (starting with /), it is relative to the top source directory.__ ''

It's unnecessary to add additional `..` to file path, however it's correct.

Comment (indie) [reply doc/sphinx/source/python_api/index.rst]: Being consistent with the rest of the documentation is important

**Comment by indie** [comment]
@postrational :   The blank line is necessary or the documentation build throws an error locally and fails to import the example.py correctly due to the `literalinclude`

**Comment by indie** [comment]
I tried again and it's okay.  2 blank lines were on the original PR even though only one shows up on Github

**Pull request merged**